modules/lb_repeat_schedules/config/install/block_content.type.lb_repeat_schedules.yml

```
@@ -0,0 +1,8 @@
+uuid: faf2c9f2-b03c-4cf6-9904-25bc84fe329e
+langcode: en
+status: true
+dependencies: {  }
+id: lb_repeat_schedules
+label: 'LB Repeat Schedules'
```

```suggestion
label: 'Repeat (Group) Schedules'
```
Other blocks don't use "LB" in their label, so I don't think there's a need here.